index.js

```diff
@@ -1,11 +1,29 @@
-function scuberGreetingForFeet(){
-  // Write your code here!
+function scuberGreetingForFeet(ride){
+  // Write your code here!  
+  if (ride <= 400){
+    return 'This one is on me!';
+  } else if (ride >= 400 && ride <= 2000){
```

Comment:
You can exclude `ride >= 400 && ` since it's already covered by the first part of the conditional. 
```suggestion
  } else if (ride <= 2000){
```